backend/src/server.py

```diff
@@ -365,6 +365,26 @@ async def list_ncnn_gpus(_request: Request):
         return json([])
 
 
+@app.route("/listgpus/nvidia", methods=["GET"])
+async def list_nvidia_gpus(_request: Request):
+    """Lists the available GPUs for NCNN"""
+    await nodes_available()
+    try:
+        # pylint: disable=import-outside-toplevel
+        from gpu import NvidiaHelper, nvidia_is_available
+
+        if not nvidia_is_available:
+            return json([])
+
+        nv = NvidiaHelper()
```

Comment:
Should we really create a new helper each time? Wouldn't it be better to just reuse a single instance? I'm just worried that the nvml `init` and `shutdown` functions might be expensive.

Reply:
Yeah you're right